recipes-core/images/orangepi-image-initramfs.bb

```diff
@@ -0,0 +1,23 @@
+DESCRIPTION = "Small image capable of booting the orangepi R1+"
```

Comment:
Should we add 6.4 to the image name as we did with the version 5?

Reply:
Image usually don't have a version in the file name, these are exceptions. It was the kernel recipe that had the 5.X version in the name, and now we have linux-orangepi_6.4.bb